scripts/zones/Northern_San_dOria/TextIDs.lua

```diff
@@ -2,122 +2,122 @@
 
 -- General Texts
     ITEM_CANNOT_BE_OBTAINED =  6589; -- Come back after sorting your inventory.
-              ITEM_OBTAINED =  6985; -- Obtained: <<<Unknown Parameter (Type: 80) 1>>><<<Possible Special Code: 01>>><<<Possible Special Code: 05>>>
+              ITEM_OBTAINED =  6986; -- Obtained: <<<Unknown Parameter (Type: 80) 1>>><<<Possible Special Code: 01>>><<<Possible Special Code: 05>>>
                GIL_OBTAINED =  6596; -- Obtained <<<Numeric Parameter 0>>> gil.
```

Comment:
double check the ID's here that the tool says did not move that are nearby ones that did - I've been having recurring issues here where it gets confused by multiple similar messages.

Reply:
<thing type="DialogTableEntry">
    <field name="index">6986</field>
    <field name="text">Obtained: ≺Numeric Parameter 0≻ guild points.≺Prompt≻
</field>

so. . .that looks wrong.

Reply:
these are usually sequential with any existing gap staying the same relative offset, so if one didn't change the others near changing is fishy (keyitem obtained moving by 1 while gil didn't = fishy). just a general thing to watch out for that has been especially prevalent in sandy zones